package.json

```
@@ -81,7 +89,10 @@
   },
   "tap": {
     "esm": false,
-    "test-ignore": "/(^|/)cli-tests-[0-9]+/",
+    "test-ignore": [
+      "/(^|/)cli-tests-[0-9]+/",
+      "test/test.mjs"
+    ],
```

Unfortunately this is not an array (see test failure under node.js 8 / 10 where test/test.mjs is run).
```suggestion
    "test-ignore": "test-ignore": "/((^|/)cli-tests-[0-9]+|\\.mjs$)/",,
```

---

Sorry apparently I broke the JSON.  I'd love to blame github UI but that's probably not reasonable.

---

@MylesBorins I'm noticing now the exclude pattern ends up in some of the snapshots so they probably have to be regenerated.